src/main/io/serial_msp.c

```diff
@@ -572,7 +575,8 @@ static uint32_t packFlightModeFlags(void)
         IS_ENABLED(FLIGHT_MODE(SONAR_MODE)) << BOXSONAR |
         IS_ENABLED(ARMING_FLAG(ARMED)) << BOXARM |
         IS_ENABLED(IS_RC_MODE_ACTIVE(BOXBLACKBOX)) << BOXBLACKBOX |
-        IS_ENABLED(FLIGHT_MODE(FAILSAFE_MODE)) << BOXFAILSAFE;
+        IS_ENABLED(FLIGHT_MODE(FAILSAFE_MODE)) << BOXFAILSAFE |
+        IS_ENABLED(FLIGHT_MODE(BOXAIRMODE)) << BOXAIRMODE;
```

Comment:
this is wrong @borisbstyle 

it should be `IS_ENABLED(IS_RC_MODE_ACTIVE(BOXAIRMODE))`